src/appleseed-max-impl/appleseedoslplugin/oslparamdlg.cpp

```diff
@@ -123,61 +115,76 @@ void OSLParamDlg::add_ui_parameter(
     const MaxParam&         max_param,
     PageGroupMap&           pages)
 {
+    const int col_1 = 10;
+    const int col_2 = 85;
+    const int col_3 = 122;
+    const int col_4 = 159;
+    const int label_width = 76;
```

Comment:
Constants are spelled in CamelCase. Also, for `col_1` to `col_4`, are these abscissa? Maybe `Col1X` then?